client/app/views/index.js

```diff
@@ -4,43 +4,43 @@ var _ = require('lodash');
 var h = require('../helpers/helpers');
 var renderPipeline = require('./components/pipelineView').render;
 
-var appendError = function appendError(err) {
-  h.$append('#content-container', 'Fetching pipelines has a problem. Please try again');
-  h.trace(err);
+var appendError = function(message) {
+  var errorMsg = message ||  'An error occurred loading the page. Please try again.';
+  return function appendError(err) {
+    h.$append('#content-container', errorMsg);
+    h.trace(err);
+  };
 };
 
 var fetchPipelinesAsync = function fetchPipelinesAsync() {
```

Comment:
I'm trying to get used to named function expressions (nfe) as it really (really) helps in debugging and logging. Also there are a couple of JS gurus (e.g. @getify and @kangax) advocating for this approach now that IE is not in the way of NFEs anymore.